theseus_gui/src/helpers/theme.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Sets the dark-mode class on <html> if dark mode is on.
+ * @param {Boolean} isDarkMode Bool value indicating if dark mode is on.
+ */
+export const toggleTheme = (isDarkMode) => {
+  if (isDarkMode) document.getElementsByTagName('html')[0].classList.add('dark-mode')
+  else document.getElementsByTagName('html')[0].classList.remove('dark-mode')
```

Comment:
here it should add the `light-mode` class

Reply:
(as if it is not dark theme it will be light theme)

Reply:
Updated. We can implement some constant values down the road if/when we want to implement oled theming and such.